app/sdil/filters/SdilFilters.scala

```diff
@@ -52,13 +54,7 @@ class SdilFilters(
       cookieCryptoFilter: SessionCookieCryptoFilter,
       sessionTimeoutFilter: SessionTimeoutFilter,
       cacheControlFilter: CacheControlFilter,
-      mdcFilter: MDCFilter
-    ) {
-  override val filters: Seq[EssentialFilter] = {
```

Comment:
can we put this override back?

Reply:
we don't require this any more as new filters contains this logic